src/onelogin/saml2/utils.py

```diff
@@ -434,7 +434,7 @@ def generate_unique_id():
 
     @staticmethod
     def parse_time_to_SAML(time):
-        """
+        r"""
```

Comment:
That r in the comment is because it contains a regular expression?

Reply:
Is ok then to add an r in the description of a method?

Reply:
src/onelogin/saml2/utils.py:446:-151: W605 invalid escape sequence '\.'
src/onelogin/saml2/utils.py:461:-228: W605 invalid escape sequence '\.'